### PR TITLE
fix(cli): bridge network + errors fix

### PR DESCRIPTION
**Pull request by alanmarkoTrilitech**
# Context

https://app.asana.com/0/1205770721173533/1206712613344550
https://app.asana.com/0/1205770721173533/1206712613344544
https://app.asana.com/0/1205770721173533/1206703424903095

# Description
These PR fixes several issues with jstz bridge - requiring -n dev flag,supporting bootstrap aliases, being able to deposit to a bootstrap account. It additionally improves error messages and does a small refactoring.

# Manually testing the PR

```
jstz bridge depost --from bootstrap1 --to alan --amount 1000000
```

### CODE REVIEW

File: crates/jstz_cli/src/bridge/deposit.rs [hunk 1]
```diff
@@ -14,18 +16,52 @@ pub fn exec(
...
+    // Check if trying to deposit to a bootsrap account.
+    if let Some(bootstrap_account) = SANDBOX_BOOTSTRAP_ACCOUNTS
+        .iter()
+        .find(|account| account.address == to_pkh.to_string())
+    {
+        bail_user_error!(
+            "Cannot deposit to the bootstrap account '{}'.",
+            bootstrap_account.address
+        );
+    }
```
Comment (johnyob) [inline crates/jstz_cli/src/bridge/deposit.rs:39]: Not too sure this is required

**Review by johnyob — COMMENTED** [review]
Added a review commit, changes lgtm mostly.

File: crates/jstz_cli/src/bridge/deposit.rs (hunk 1 above)
Comment (alanmarkoTrilitech) [reply crates/jstz_cli/src/bridge/deposit.rs:39]: I think it is - it clearly shouldn't be possible to bridge to a bootstrap account. 

We might want to have something where the bridge contract declines it automatically. Anyway, the bridge will be reworked quite a lot in the future - e.g. added withdrawals, tokens etc.

**Review by alanmarkoTrilitech — COMMENTED** [review]
The review commit looks good to me. YOu have to approve the PR though.

**Comment by johnyob** [comment]
> I think it is - it clearly shouldn't be possible to bridge to a bootstrap account.

These are different kinds of accounts. Bridging to a bootstrap account might be desirable -- keeping the same bootstrap account addresses across Tezos and jstz. 

> We might want to have something where the bridge contract declines it automatically. Anyway, the bridge will be reworked quite a lot in the future - e.g. added withdrawals, tokens etc.

Agreed, I won't block this based on this.

**Review by johnyob — APPROVED** [review]
LGTM 🚢  -- please squash the review commit before merging 🚀

**Comment by alanmarkoTrilitech** [comment]
> > I think it is - it clearly shouldn't be possible to bridge to a bootstrap account.
> 
> These are different kinds of accounts. Bridging to a bootstrap account might be desirable -- keeping the same bootstrap account addresses across Tezos and jstz.
> 
> > We might want to have something where the bridge contract declines it automatically. Anyway, the bridge will be reworked quite a lot in the future - e.g. added withdrawals, tokens etc.
> 
> Agreed, I won't block this based on this.

thanks for not blocking it, yeah it seems like a design decision that should require a bit of thought - for example it might be also useful to have L1/L2 addresses all disjoint

**Pull request merged**